File: admin-api/java-api/src/main/java/net/lab1024/smartadmin/module/support/captcha/domain/CaptchaVO.java
package net.lab1024.smartadmin.module.support.captcha.domain;

import io.swagger.annotations.ApiModelProperty;
import lombok.Data;

/**
 * 图形验证码 VO
 *
 * @author 胡克
 * @date 2021/8/31 20:52
 */
@Data
public class CaptchaVO {

    @ApiModelProperty("验证码唯一标识" )
    private String captchaUuid;

    @ApiModelProperty("验证码Base64图片" )
    private String captchaBase64Image;
}

File: admin-web/typescript-element-plus-vue/src/api/system/login/model/captcha-vo.ts
/*
 * @Author: zhuoda
 * @Date: 2021-08-03 22:27:11
 * @LastEditTime: 2022-01-07
 * @LastEditors: zhuoda
 * @Description:
 */

/**
 *
 * @export
 * @interface CaptchaVO
 */
export interface CaptchaVO {
  /**
   * 验证码唯一标识
   * @type {string}
   * @memberof CaptchaVO
   */
  captchaUuid: string;
  /**
   * 验证码Base64图片
   * @type {string}
   * @memberof CaptchaVO
   */
  captchaBase64Image: string;
}
